apps/synt-vedtakshistorikk-service/src/main/java/no/nav/testnav/apps/syntvedtakshistorikkservice/service/ArenaForvalterService.java

```diff
@@ -198,4 +217,14 @@ public void slettArbeidssoekerIArena(String ident, String miljoe) {
             arenaForvalterConsumer.slettBrukerIArenaForvalteren(ident, miljoe);
         }
     }
+
+    public NyttVedtakTiltak finnTiltak(String personident, String miljoe, NyttVedtakTiltak tiltaksdeltakelse) {
+        var response = arenaForvalterConsumer.finnTiltak(getFinnTiltakRequest(personident, miljoe, tiltaksdeltakelse));
+        if (nonNull(response) && !response.getNyeRettigheterTiltak().isEmpty()) {
```

Comment:
Et bra pattern synes jeg er å legge til spesifikke gettere for collections i bean-klasser. Den spesifikke getter tester om collection, f.eks liste er null og hvis den er initieres den med tomt array før verdi returneres.

Da slipper du å null-teste en collection, men kan skrive

```
if (!response.getNyeRettigheterTiltak().isEmpty()) {
}
```
i stedet.

Reply:
Veldig god ide, skal fikse :)